src/Apps/Order/Components/BankAccountPicker.tsx

```diff
@@ -107,7 +128,7 @@ export const BankAccountPicker: FC<Props> = props => {
 
               return (
                 <BorderedRadio value={internalID} key={internalID}>
-                  <BankDebitDetails last4={last4} />
+                  <BankDebitDetails last4={last4!} />
```

Comment:
Why is this non-null assertion necessary given the `BankAccountRecord` interface?

Reply:
it's actually not, good catch! will remove.